opba-consent-rest-api/src/main/resources/static/tpp_consent_api.yml

```diff
@@ -226,7 +187,7 @@ paths:
       security:
         - redirectCookie: []
 
-  /v1/consent/{auth-id}/fromAspsp/{redirectState}/nok/{fromAspspRedirectCode}:
+  /v1/consent/{auth-id}/fromAspsp/nok/{redirectState}:
```

Comment:
Suggest to rephrase endpoint to:
`/v1/consent/{auth-id}/fromAspsp/nok/{redirectState}` -> `/v1/consent/{auth-id}/fromAspsp/{redirectState}/nok`

Reply:
endpoint rephrased